### PR TITLE
Update microsoft-365-compliance-center-permissions.md

### PR DESCRIPTION
**Pull request by msbemba**
Made changes to permissions in the Microsoft purview compliance portal section and reworded it to reduce ambiguity.

Fixes#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/11121

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/microsoft-365-compliance-center-permissions.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ ms.custom:
...
 
-To view the **Permissions** tab in the compliance portal, users need to be a global administrator or need to be assigned the *Role Management* role (a role is assigned only to the *Organization Management* role group). The *Role Management* role allows users to view, create, and modify role groups.
+To view the **Permissions** tab in the compliance portal, users need to be a global administrator or need to be assigned the *Role Management* role (this role is assigned only to the Organization Management role group by default). The *Role Management* role allows users to view, create, and modify role groups.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/microsoft-365-compliance-center-permissions.md]:
```suggestion
To view the **Permissions** tab in the compliance portal, users need to be a global administrator or need to be assigned the *Role Management* role (this role is assigned only to the *Organization Management* role group by default). The *Role Management* role allows users to view, create, and modify role groups.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@robmazz Please merge this Pr. Thanks

**Comment by yogkumgit** [comment]
@robmazz Gentle reminder.

**Comment by yogkumgit** [comment]
Closing as per the comment in #11121

**Pull request closed without merging**